common/test/acceptance/tests/lms/test_lms_course_discovery.py

```diff
@@ -60,10 +60,10 @@ def test_search(self):
         Make sure you can search for courses.
         """
         self.page.visit()
-        self.assertEqual(len(self.page.result_items), 12)
+        self.assertEqual(len(self.page.result_items), 36)
```

Comment:
What is the reason for increase in these numbers?

Reply:
I don't know the real reason behind it, in this test each result item increased by 3 times.

Reply:
We should investigate to find out the reason.